packages/tasks_panel/test/misc/e2e-config.spec.ts

```diff
@@ -0,0 +1,472 @@
+import { SinonMock, SinonSandbox, createSandbox } from "sinon";
+import {
+  LabelType,
+  ProjectTypes,
+  addTaskDefinition,
+  areResourcesReady,
+  collectProjects,
+  generateMtaDeployTasks,
+  isFileExist,
+  isTasksSettled,
+  waitForResource,
+} from "../../src/misc/e2e-config";
+import { expect } from "chai";
+import { MockVSCodeInfo, resetTestVSCode, testVscode } from "../utils/mockVSCode";
+import * as utils from "../../src/utils/task-serializer";
+import { afterEach } from "mocha";
+import { concat, extend, last, split } from "lodash";
+import * as cfTools from "@sap/cf-tools/out/src/cf-local";
+import * as path from "path";
```

Comment:
subjective style: it is generally better imho to only import the specific functions you need, not `*`